code/game/objects/structures/guillotine.dm

```diff
@@ -137,53 +137,49 @@
 	blade_status = GUILLOTINE_BLADE_RAISED
 	icon_state = "guillotine_raised"
 
-/// Drops the guillotine blade, potentially beheading or harbing the buckled mob
+/// Drops the guillotine blade, potentially beheading or harming the buckled mob
 /obj/structure/guillotine/proc/drop_blade(mob/user)
 	if (has_buckled_mobs() && blade_sharpness)
-		var/mob/living/carbon/human/H = buckled_mobs[1]
+		var/mob/living/carbon/human/victim = buckled_mobs[1]
 
-		if (!H)
+		if (!victim)
 			return
 
-		var/obj/item/bodypart/head/head = H.get_bodypart("head")
-
-		if (QDELETED(head))
-			return
+		var/obj/item/bodypart/head/head = victim.get_bodypart("head")
 
 		playsound(src, drop_sound, 100, TRUE)
-		if (blade_sharpness >= GUILLOTINE_DECAP_MIN_SHARP || head.brute_dam >= 100)
-			head.dismember()
-			log_combat(user, H, "beheaded", src)
-			H.regenerate_icons()
-			unbuckle_all_mobs()
-			kill_count += 1
-
-			var/blood_overlay = "bloody"
-
-			if (kill_count == 2)
-				blood_overlay = "bloodier"
-			else if (kill_count > 2)
-				blood_overlay = "bloodiest"
-
-			blood_overlay = "guillotine_" + blood_overlay + "_overlay"
-			cut_overlays()
-			add_overlay(mutable_appearance(icon, blood_overlay))
-
-			// The crowd is pleased
-			// The delay is to making large crowds have a longer laster applause
-			var/delay_offset = 0
-			for(var/mob/M in viewers(src, 7))
-				var/mob/living/carbon/human/C = M
-				if (ishuman(M))
-					addtimer(CALLBACK(C, TYPE_PROC_REF(/mob/, emote), "clap"), delay_offset * 0.3)
+		if(head)
```

Comment:
```suggestion
		if(QDELETED(head))
			return
			
		// rest of the code will now be on this indentation
```

Reply:
This breaks the guillotine code entirely...

Reply:
alright

Reply:
After the double-check it appears that i'm just being schizofrenical. Nevermind. But the issue about beheaded bodies is still in-tact.

Reply:
This still doesn't allow to use guillotine on beheaded people proparly.

Reply:
The thing is, i think, that return just discards all the next steps that should be there if there is no head and it's just becomes useless object that can't be even usable. And i kinda don't know to what should it return.

Reply:
oh this proc actually has a return value? i didn't see that. it's okay then, maybe another day.